tests/api.c

```diff
@@ -14731,6 +14736,156 @@ static void test_wc_PKCS7_EncodeEncryptedData (void)
 } /* END test_wc_PKCS7_EncodeEncryptedData() */
 
 
+/* Testing wc_SignatureGetSize() */
+static int test_wc_SignatureGetSize(void)
+{
+
+    int ret = 0; 
+    enum wc_SignatureType sig_type;
+    word32 key_len;
+
+    /* Initialize ECC Key */
+    #if defined(HAVE_ECC) && !defined(NO_ECC256)
+        ecc_key ecc; 
+        
+        const char* qx =
+            "fa2737fb93488d19caef11ae7faf6b7f4bcd67b286e3fc54e8a65c2b74aeccb0";
+        const char* qy = 
+            "d4ccd6dae698208aa8c3a6f39e45510d03be09b2f124bfc067856c324f9b4d09";
+        const char* d = 
+            "be34baa8d040a3b991f9075b56ba292f755b90e4b6dc10dad36715c33cfdac25";
+    
+        ret = wc_ecc_init(&ecc);
+        if (ret != 0) {
+            ret = WOLFSSL_FATAL_ERROR;
+            goto done;
```

Comment:
Our coding standards prefer us not to use "goto" when possible.  Can you re-work your use of goto's using a different error propagation technique?

It has been common practice in the unit tests to separate blocks of code with checks against "ret", for example:

```
ret = <function_call>;
if (ret == 0) {
    /* code block here */
    ret = <function_call>;
}

if (ret == 0) {
    /* code block here */
    ret = <function_call>;
}

/* cleanup done at the end */
```